newdle/client/src/components/DurationPicker.js

```diff
@@ -0,0 +1,40 @@
+import React, {useState} from 'react';
+import PropTypes from 'prop-types';
+import _ from 'lodash';
+import {Dropdown} from 'semantic-ui-react';
+import styles from './DurationPicker.module.scss';
+
+function getDurationOptions(min, max, interval) {
+  return _.range(min, max + interval, interval).map(i => ({key: i, value: i, text: i}));
+}
+
+export default function DurationPicker({minDuration, maxDuration, interval}) {
+  const [durationValue, setDurationValue] = useState(minDuration);
+  const durationOptions = getDurationOptions(minDuration, maxDuration, interval);
```

Comment:
This should probably be a fully controlled component. The state is not of much use inside the current component as we will have to use it further into the process.

Reply:
Indeed! IIRC. I didn't comment on that one since until we integrate it with the rest having local state is probably easier.

Reply:
The `Timeline` is already merged and since this will be integrated with the `Timeline` I can change it now.